src/main/java/dukeexecpetions/EventEmptyException.java

```diff
@@ -0,0 +1,26 @@
+package dukeexecpetions;
+
+/** Exception when an empty field is passed for a field*/
+public class EventEmptyException extends EmptyFieldException{
+    /** text for when from entry is missing */
+    private static final String missing_from_txt = "from - What time does %s start from?\n";
```

Comment:
Could this variable name be treated as a constant, and thus be named using SCREAMING_SNAKE_CASE?